go-controller/pkg/util/net.go

```diff
@@ -317,11 +317,10 @@ func IPsToNetworkIPs(ips ...*net.IPNet) []*net.IPNet {
 	return nets
 }
 
-// StringsToIPs takes a slice of strings and returns a slice of net.IPs
-func StringsToIPs(ips []string) []net.IP {
-	s := make([]net.IP, len(ips))
-	for i := range ips {
-		s[i] = net.ParseIP(ips[i])
+func IPNetsIPToString(ips []*net.IPNet) []string {
+	ipAddrs := make([]string, 0)
```

Comment:
Well, if you have the chance `IPNetsIPToStringSlice` lol I guess I am very inconsistent with naming as well

Reply:
haha lol its ok its not a hard change to do :)

Reply:
done.